test/test_topology.py

```diff
@@ -52,13 +52,13 @@ def get_topology_type(self):
 def create_mock_topology(
         seeds=None,
         replica_set_name=None,
-        monitor_class=DummyMonitor):
+        monitor_class=DummyMonitor, direct_connection=False):
```

Comment:
This line should match the surrounding code style where each argument is on a new line.

Reply:
Done.